shared/fs/filepreview/av-view.native.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+import * as React from 'react'
+import {globalColors} from '../../styles'
+import WebView from '../../common-adapters/web-view.native'
```

Comment:
Is there a reason not to have a single adapter for both desktop and native webviews? They seem to have the same API, so we might as well make them the same. Would avoid having to have this `.native.js` file.

Reply:
They actually have different APIs. They accept different forms of parameters to specify source URI; they also use different interface for injecting JavaScript/CSS. But I guess we can still unify them by shipping our own Props. Will do; thanks!